containers/dapr-dotnetcore-3.0/README.md

```diff
@@ -0,0 +1,120 @@
+# Dapr with C# (.NET Core 3.0)
+
+## Summary
+
+*Develop Dapr applications using C# and .NET Core 3.0. Includes all needed SDKs, extensions, and dependencies.*
+
+| Metadata | Value |  
+|----------|-------|
+| *Contributors* | The VS Code Team |
```

Comment:
Definitely take credit for the hard work by adding yourself here. The VS Code team itself does not have expertise in Dapr, so we shouldn't take the credit.  😊

Reply:
Updated.